app/assets/javascripts/discourse/widgets/post-cooked.js.es6

```diff
@@ -169,6 +169,12 @@ export default class PostCooked {
               "fast",
               finished
             );
+          } else if (e.jqXHR.status === 403) {
+            $blockQuote.showHtml(
+              $(`<div class='expanded-quote'>${iconHTML("lock")}</div>`),
+              "fast",
+              finished
+            );
```

Comment:
Perhaps something like this to avoid duplicating the code above?

```
if ([403, 404].includes(e.jqXHR.status)) {
  const icon = e.jqXHR.status === "403" ? "lock" : "trash-o"; 

  $blockQuote.showHtml(
    $(`<div class='expanded-quote'>${iconHTML(icon)}</div>`),
    "fast",
    finished
  );
}
```